pages/order-overview/index.js

```diff
@@ -1,22 +1,31 @@
-export default function OrderOverview({ items }) {
+export default function OrderOverview({ items, show }) {
   function getTotalPrice(items) {
     return items.reduce((memo, item) => memo + item.price * item.amount, 0);
   }
+
+  const drawerClasses = ["side-drawer"];
+
+  if (show) {
+    drawerClasses.push("open");
+  }
+
```

Comment:
You may remove these lines as well. They try to apply styling via css classes.
Since we're using `styled-components` you'd be styling your elements directly via `styled` and not the `classname`.

Reply:
so I have to think which component gets what styling from the global styling? can u make like an example please